Sources/lite/main.swift

```diff
@@ -58,7 +58,7 @@ func runBehaviorTests() -> Bool {
                                pathExtensions: ["js"],
                                testDirPath: "Tests/BehaviorTests",
                                testLinePrefix: "//",
-                               parallelismLevel: .none,
```

Comment:
Was this included by mistake? I thought it worked worse when we tried it at higher parallelism levels.

Reply:
Indeed!